fetch.bs

```diff
@@ -5156,17 +5244,19 @@ constructor must run these steps:
    <li><p>Set <var>request</var>'s <a for=request>referrer</a> to
    "<code>client</code>"
 
-   <li><p>Set <var>request</var>'s
-   <a for=request>referrer policy</a> to the empty string.
+   <li>
+    <p>Set <var>request</var>'s <a for=request>referrer policy</a> to the empty string.
+
+    <p class=note>This is done to ensure that when a service worker "redirects" a request, e.g.,
+    from an image in a cross-origin stylesheet, and makes modifications, it no longer appears to
+    come from the original source (i.e., the cross-origin stylesheet), but instead from the service
+    worker that "redirected" the request. This is important as the original source might not even be
+    able to generate the same kind of requests as the service worker. Services that trust the
+    original source could therefore be exploited were this not done, although that is somewhat
+    farfetched.
```

Comment:
Why did you move this note by the way? It really applies to the whole "if any of init's member are present" clause.